pyproject.toml

```diff
@@ -221,10 +222,49 @@ select = ["I", "E", "W", "D", "F", "PL", "NPY", "N", "PERF", "RUF", "UP"]
 exclude = ["docs/conf.py", "setup.py"]
 
 ignore = [
-  "D407", "D203", "D213", "D416", "PLR0912", "PLR0911", "PLR0915", "PLR2004", "PLR0913",  "N802",
-  "D103", "D100", "D101", "D102", "D105", "D104", "D107", "D205", "D301","D400", "D401", "D404", "D415", "D417", "E501",
-  "PLR1714", "PLC0414", "PLC0105", "PLW2901", "PERF203", "PERF401", "RUF017", "RUF009", "RUF005", "RUF015", "RUF002", "RUF003", "N806", "N817",
-  "N818", "N803", "UP038", "UP035"
+  "D407",
+  "D203",
+  "D213",
+  "D416",
+  "PLR0912",
+  "PLR0911",
+  "PLR0915",
+  "PLR2004",
+  "PLR0913",
+  "N802",
+  "D103",
+  "D100",
+  "D101",
+  "D102",
+  "D105",
+  "D104",
+  "D107",
+  "D205",
+  "D301",
+  "D400",
+  "D401",
+  "D404",
+  "D415",
+  "D417",
+  "E501",
+  "PLR1714",
+  "PLC0414",
+  "PLC0105",
+  "PLW2901",
+  "PERF203",
+  "PERF401",
+  "RUF017",
+  "RUF009",
+  "RUF005",
+  "RUF015",
+  "RUF002",
+  "RUF003",
+  "N806",
+  "N817",
+  "N818",
+  "N803",
+  "UP038",
+  "UP035",
```

Comment:
That is quite inconvenient. I could not find a way to avoid reformatting that part automatically. I prefer the original formatting, way more readable.

Reply:
Just reverted.